### PR TITLE
AP1587 geckoboard nonpassported

### PR DESCRIPTION
**Pull request by Obsiye**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-1587)

Show the total number per day split by passported vs. non-passported submitted applications on geckoboard

- Update the existing /widget_data_providers/applications.rb with nonpassported and passported data

- Create tests

The dataset is just added ontop of our existing 'Applications' widget 
image

## Checklist

Before you ask people to review this PR:

- [x] Tests and rubocop should be passing: `bundle exec rake`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- [x] The PR description should say what you changed and why, with a link to the JIRA story.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/services/test_application_creation_service.rb [hunk 1]
```diff
@@ -1,13 +1,6 @@
 class TestApplicationCreationService
   APPLICATION_TEST_TRAITS = %i[
-    at_initiated
     at_checking_applicant_details
-    at_checking_passported_answers
-    at_applicant_details_checked
-    awaiting_applicant
-    applicant_entering_means
-    provider_entering_merits
-    at_checking_merits_answers
```
Comment (mpw5) [inline app/services/test_application_creation_service.rb:10]: I might be missing something but I'm not sure I understand why these are being removed? It seems like it's removing most of the purpose of this class.

Also it doesn't look like `TestApplicationCreationService` is used by the geckoboard tests, so should these changes even be part of this PR?

File: app/services/test_application_creation_service.rb [hunk 2]
```diff
@@ -31,9 +24,14 @@ def providers
...
       trait,
-      provider: provider
+      provider: provider,
+      applicant: passported_applicant
     )
   end
+
+  def passported_applicant
+    FactoryBot.create :applicant, date_of_birth: '10-01-1980', last_name: 'Walker', national_insurance_number: 'JA293483A'
+  end
```
Comment (mpw5) [inline app/services/test_application_creation_service.rb:36]: Similar to above, I'm not sure why this change is necessary here (sorry if I'm missing something obvious 😁)

**Review by mpw5 — COMMENTED** [review]
Looks mostly good to me. I've left a comment about the changes to `test_application_creation_service`, as I'm not sure I understand the changes you've made there.

It's probably a good idea to get @colinbruce (sorry Colin) to take a look at it too as I have little experience with GeckoBoard and might be missing something

File: app/services/test_application_creation_service.rb (hunk 1 above)
Comment (Obsiye) [reply app/services/test_application_creation_service.rb:10]: sorry that was leftover stuff. i'll remove all changes to that file

**Review by colinbruce — CHANGES_REQUESTED** [review]
Couple of possible updates needed... happy to discuss if I've missed some reasoning

**Review by colinbruce — APPROVED** [review]
:+1:
